### ISSUE TITLE
problem when testing

### ISSUE THREAD

**Issue by arcobaleno1996** [HaleyPei/Implement-of-SCA-CNN#1]
Hi, I've tried your code of implement of sca-cnn, but get a very low bleu4 score about 0.05 in coco dataset. Is it complete or any  bugs in your networks? Looking forward your reply!

**Comment by TangZwei** [comment]
do you slove it? when validating each epoch, i find bleu4 is very low.

**Comment by arcobaleno1996** [comment]
no, but I found another implement of sca-cnn in https://github.com/stevehuanghe/image_captioning which you need to modify the dimension of lstm when decoding

**Comment by Sun-WeiZhen** [comment]
> no, but I found another implement of sca-cnn in https://github.com/stevehuanghe/image_captioning which you need to modify the dimension of lstm when decoding

Have you implemented this code about sca-cnn in                                                                         https://github.com/stevehuanghe/image_captioning.I had a lot of questions in the verification process.Could you please provide the verification code about coco-caption

**Comment by hhBonnie** [comment]
> no, but I found another implement of sca-cnn in https://github.com/stevehuanghe/image_captioning which you need to modify the dimension of lstm when decoding

您好，请问您在运行eval.py时有遇到AttributeError: 'DecoderWithAttention' object has no attribute 'attention'的问题吗？请问您是怎么解决的呢？
